general/community/meetings/202402.md

```diff
@@ -10,7 +10,7 @@ sidebar_position: -202402
 
 16th February 2024 - Time 12:00PM UTC
 
-[Meeting Link](https://moodle.org/mod/bigbluebuttonbn/view.php?id=8596)
+[Meeting Recording](https://recordings.rna1.blindsidenetworks.com/moodlehq/e9b35f3e7056aa2378e3076420f2f69ba6e0cd9e-1708084172089/capture/)

```

Comment:
I just tried going to this link, and it did not work fo me:


> Error 404
> The recording could not be loaded
> 
> The recording that you are attempting to access cannot be loaded. This may be a protected recording. If so, use the authenticated recording link from your LMS system.
> 
> Contact support if you are still having issues.